frontend/src/app/[locale]/search/page.tsx

```diff
@@ -48,6 +49,8 @@ function Search({ searchParams, params }: SearchPageProps) {
     resolvedSearchParams.page = "1";
   }
 
+  const searchResultsPromise = searchForOpportunities(convertedSearchParams);
```

Comment:
moved out here to allow the filter components to consume the response to get facets